runtime/themes/adwaita-dark.toml

```diff
@@ -0,0 +1,170 @@
+# Author: Mofiqul Islam <mofi0islam@gmail.com>
+
+"attribute" = "orange_4"
+
+"type" = "teal_2"
+"type.builtin" = "teal_2"
+
+"constructor" = "blue_2"
+
+"constant" = "violet_2"
+"constant.builtin" = { fg = "violet_2", modifiers = ["bold"] }
+"constant.character" = "teal_3"
+"constant.numeric" = { fg = "teal_3", modifiers = ["bold"] }
+"constant.character.escape" = "violet_2"
+
+"string" = "teal_2"
+"string.regexp" = "purple_2"
+"string.special" = "blue_2"
+
+"comment" = "dark_2"
+
+"variable" = "light_4"
+"variable.parameter" = "orange_2"
+"variable.builtin" = "orange_2"
+"variable.other" = "teal_2"
+"variable.other.member" = "teal_2"
+
+"label" = "purple_2"
+
+"punctuation" = "light_4"
+"punctuation.delimiter" = "light_4"
+"punctuation.bracket" = "light_4"
+"punctuation.special" = "red_3"
+
+"keyword" = { fg = "orange_2", modifiers = ["bold"] }
+"keyword.control" = { fg = "orange_2", modifiers = ["bold"] }
+"keyword.operator" = "purple_2"
+"keyword.directive" = { fg = "orange_2", modifiers = ["bold"] }
+"keyword.function" = "orange_2"
+"keyword.storage" = { fg = "orange_2", modifiers = ["bold"] }
+
+"operator" = "purple_2"
+
+"function" = "blue_2"
+"function.builtin" = "blue_2"
+"function.macro" = { fg = "blue_2", modifiers = ["bold"] }
+"function.special" = { fg = "blue_2", modifiers = ["bold"] }
+
+"tag" = "teal_2"
+
+"namespace" = "orange_2"
+
+"markup" = "light_4"
+"markup.heading" = { fg = "teal_2", modifiers = ["bold"] }
+"markup.list" = { fg = "orange_2", modifiers = ["bold"] }
+"markup.bold" = { fg = "light_4", modifiers = ["bold"] }
+"markup.italc" = { fg = "light_4", modifiers = ["italic"] }
+"markup.link" = { fg = "blue_3", modifiers = ["underlined"] }
+"markup.quote" = { fg = "light_3", modifiers = ["italic"] }
+"diff.plus" = "teal_3"
+"diff.minus" = "red_1"
+"diff.delta" = "orange_3"
+"diff.delta.moved" = "orange_2"
+
+"ui.background" = { fg = "light_4", bg = "libadwaita_dark" }
+"ui.background.separator" = { fg = "split_and_borders", bg = "libadwaita_dark" }
+"ui.cursor" = { fg = "libadwaita_dark", bg = "light_5" }
+"ui.cursor.insert" = { fg = "libadwaita_dark", bg = "light_5" }
+"ui.cursor.select" = { fg = "libadwaita_dark", bg = "light_5" }
+"ui.cursor.match" = { fg = "libadwaita_dark", bg = "blue_2" }
+"ui.cursor.primary" = { fg = "libadwaita_dark", bg = "light_7" }
+"ui.linenr" = "dark_2"
+"ui.linenr.selected" = { fg = "light_7", bg = "libadwaita_dark_alt", modifiers = [
+  "bold",
+] }
+"ui.statusline" = { fg = "light_4", bg = "libadwaita_dark_alt" }
+"ui.statusline.inactive" = { fg = "light_4", bg = "libadwaita_dark_alt" }
+"ui.statusline.insert" = { fg = "light_4", bg = "teal_4" }
+"ui.statusline.select" = { fg = "light_4", bg = "blue_4" }
+"ui.popup" = { bg = "libadwaita_popup" }
+"ui.window" = "split_and_borders"
+"ui.help" = { bg = "libadwaita_dark_alt" }
+"ui.text" = "light_4"
+"ui.virtual.ruler" = "dark_1"
+"ui.virtual.indent-guide" = "dark_1"
```

Comment:
You may also want to style `ui.virtual.whitespace` (use `:set whitespace.render all` to enable whitespace rendering temporarily)

Reply:
The soft-wrap indicator is controlled with `ui.virtual.wrap`: https://docs.helix-editor.com/master/themes.html#interface

It would be good to style that here as well. You can have all of the keys under `ui.virtual` use the same color by only styling `ui.virtual`:

```toml
"ui.virtual" = "dark_1"
```

Reply:
Shall I remove all of these?
```
"ui.virtual.ruler" = "dark_1"
"ui.virtual.indent-guide" = "dark_1"
"ui.virtual.whitespace" = "dark_1"
```
And do only 
`"ui.virtual" = "dark_1"`

Reply:
Yep that would work - those `ui.virtual.*` scopes fall back to `ui.virtual` if they aren't defined, so they would all use `dark_1` with that change

Reply:
I have added it